src/components/form/DynamicForm.tsx

```diff
@@ -48,18 +43,19 @@ export interface Callbacks {
 export interface Props extends ParentProps, Callbacks {}
 @connectActionSheet
 export default class DynamicForm extends React.Component<Props, State> {
+	private formData: any = {};
+
 	state = {
-		formData: {},
 		submitStatus: '',
 		hasCameraPermission: false,
-		imageLists: [],
+		imageList: [],
 	};
 
 	async componentWillMount() {
 		let hiddenCount = 0;
-		console.log('Dynamic From: ' + this.props.formSchema);
+		// console.log('Dynamic From: ' + this.props.formSchema);
 		this.props.formSchema.map((field: any) => {
-			console.log('Field: ' + field.label);
+			// console.log('Field: ' + field.label);
```

Comment:
Can we please remove the commented out code. (Makes the code look a bit cluttered)

Reply:
Will do, thanks Nick.